src/test/java/net/sf/jabref/importer/fileformat/BiblioscapeImporterTest.java

```diff
@@ -0,0 +1,96 @@
+package net.sf.jabref.importer.fileformat;
+
+import java.io.InputStream;
+import java.util.LinkedList;
+import java.util.List;
+import net.sf.jabref.Globals;
+import net.sf.jabref.JabRefPreferences;
+import net.sf.jabref.importer.OutputPrinterToNull;
+import net.sf.jabref.importer.fileformat.BiblioscapeImporter;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+public class BiblioscapeImporterTest {
+
+    @Before
+    public void setUp() throws Exception {
+        if (Globals.prefs == null) {
+            Globals.prefs = JabRefPreferences.getInstance();
+        }
+    }
+
+    @Test
+    public void testGetFormatName() {
+        BiblioscapeImporter importer = new BiblioscapeImporter();
+        Assert.assertEquals(importer.getFormatName(), "Biblioscape");
+    }
+
+    @Test
+    public void testGetCLIID() {
+        BiblioscapeImporter importer = new BiblioscapeImporter();
+        Assert.assertEquals(importer.getCLIId(), "biblioscape");
+    }
+
+    @Test
+    public void testIsRecognizedFormat() throws Throwable {
+        BiblioscapeImporter importer = new BiblioscapeImporter();
+        LinkedList<String> list = new LinkedList<String>();
+        list.add("BiblioscapeImporterTest.txt");
+        list.add("CopacImporterTest1.txt");
+        list.add("CopacImporterTest2.txt");
+        list.add("CopacImporterTest3.txt");
+        list.add("IEEEImport1.txt");
+        list.add("IsiImporterTest1.isi");
+        list.add("IsiImporterTestInspec.isi");
+        list.add("IsiImporterTestWOS.isi");
+        list.add("IsiImporterTestMedline.isi");
+        list.add("RisImporterTest1.ris");
+        for (String str : list) {
+            Throwable throwable = null;
+            Object var6_7 = null;
+            try {
+                InputStream is = BiblioscapeImporterTest.class.getResourceAsStream(str);
+                try {
+                    Assert.assertTrue(importer.isRecognizedFormat(is));
+                } finally {
+                    if (is != null) {
+                        is.close();
+                    }
+                }
+            } catch (Throwable var6_8) {
+                if (throwable == null) {
+                    throwable = var6_8;
+                } else if (throwable != var6_8) {
+                    throwable.addSuppressed(var6_8);
+                }
+                throw throwable;
+            }
```

Comment:
1) Please use Arrays.asList instead of adding all the entries with their list.add line. 
2) please have a look at how try-with-resources work. In #310 this is implemented more concisely.